R/human_infection.R

```diff
@@ -272,28 +443,28 @@ calculate_patent_infections <- function(
   philm <- anti_parasite_immunity(
     min = parameters$philm_min, max = parameters$philm_max, a50 = parameters$alm50,
     k = parameters$klm, iaa = iaa, iam = iam)
-  patent_infections <- bitset_at(infections, bernoulli_multi_p(philm))
+  lm_det_infections <- bitset_at(infections, bernoulli_multi_p(philm))
   
   incidence_renderer(
     variables$birth,
     renderer,
-    patent_infections,
-    'inc_patent_',
-    parameters$patent_incidence_rendering_min_ages,
-    parameters$patent_incidence_rendering_max_ages,
+    lm_det_infections,
+    'inc_lm_det_',
+    parameters$lm_det_incidence_rendering_min_ages,
+    parameters$lm_det_incidence_rendering_max_ages,
     timestep
   )
   incidence_probability_renderer(
     variables$birth,
     renderer,
     infections,
     philm,
-    'inc_patent_',
-    parameters$patent_incidence_rendering_min_ages,
-    parameters$patent_incidence_rendering_max_ages,
+    'inc_lm_det_',
+    parameters$lm_det_incidence_rendering_min_ages,
+    parameters$lm_det_incidence_rendering_max_ages,
     timestep
   )
-  patent_infections
+  lm_det_infections
 }
```

Comment:
I think I may have posted a similar comment before, but I don't understand what this output would be used for? With a diagnostic like Light microscopy, the inutitive measure is one of prevalence.

Reply:
I'm happy to take this out if you don't think it's useful. I was just trying to give flexible options to the user without thinking much about how it would be used.